docs/introduction/setting-environment-variables.md

```diff
@@ -0,0 +1,74 @@
+# Setting environment variables
+
+In this article describes how to set environments variable in different environment.
+
+The most common uses are: Kubernetes, Docker or native installation.
+
+!!! note
+    
+    Bellow is descibed the easiest way. For better understanding we recomend to find out the documentation of the specific environment.
+
+!!! tip
+
+    You may want to tune your configuration via [`.env` file](https://symfony.com/doc/4.4/configuration.html#configuring-environment-variables-in-env-files).
```

Comment:
I suggest mentioning `.env.local` file as it's not committed and may come in handy on the local machine.

Reply:
mentioned